06_puppy/patrickmarabeas/mapStore.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+type MapStore struct {
```

Comment:
Would be nice to see some documentation on the MapStore struct as well, since it is a part of the exported package.